src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -61,6 +65,13 @@ private static ArgumentParser getArgumentParser() {
                 .setDefault(Optional.empty())
                 .help("The date to stop filtering.");
 
+        parser.addArgument("-formats")
+                .nargs("*")
+                .metavar("FORMAT")
+                .setDefault(DEFAULT_FORMATS)
+                .help("The file formats to process. "
+                        + "If not provided, java and adoc will be used.");
```

Comment:
Put quotes on the `java` and `adoc` to be more specific those are the file format.